dynamo/sample_data.py

```diff
@@ -4,6 +4,8 @@
 import os
 import ntpath
 
+from .dynamo_logger import main_info, main_log_time
```

Comment:
it will be great to use the main_log_time to show the downloading progress.

Reply:
main_log_time wont work and thus I will create a webhook in logger to report progress in our style.